README.md

```diff
@@ -53,7 +53,7 @@ The Open Event Server can be easily deployed on a variety of platforms. Detailed
 
 One-click Heroku and Docker deployments are also available:
 
-[![Deploy to Docker Cloud](https://files.cloud.docker.com/images/deploy-to-dockercloud.svg)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+[![Deploy to Docker Cloud](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4e/Docker_%28container_engine%29_logo.svg/150px-Docker_%28container_engine%29_logo.svg.png)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
Gives 404, so either this should be removed or fixed

Reply:
@iamareebjamal I am seeing the image on GitHub also in PR 
<img width="909" alt="Screenshot 2019-10-30 at 7 36 51 PM" src="https://user-images.githubusercontent.com/37082605/67865099-c1432480-fb4c-11e9-950e-28e62a5dea44.png">

Reply:
> https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server

@iamareebjamal The link is the same as you have sent above for after the click

Reply:
@iamareebjamal I am not getting what you are telling sorry for that as the link "https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server" opens **hub.docker.com** for me

Reply:
"And it gives 404, so it doesn't work."

Can you deploy to docker cloud through the link? 

Does it show anything else than `404 Page not found`?

If not, then the link doesn't work. There is no use of a button which doesn't work. Either it should be fixed so clicking on it deploys to docker cloud or removed

Reply:
@iamareebjamal I got it now what you are trying to say will remove it ASAP